petab/petablint.py

```diff
@@ -87,13 +88,18 @@ def parse_cli_args():
     group.add_argument(
         "-y",
         "--yaml",
+        dest="yaml_file_name_deprecated",
+        help="PEtab YAML problem filename. "
+        "*DEPRECATED* pass the file name as positional argument instead.",
+    )
+    group.add_argument(
         dest="yaml_file_name",
         help="PEtab YAML problem filename",
+        nargs="?",
```

Comment:
I wonder if it's possible to say `yaml_file_name_deprecated` takes 0 arguments, i.e. it's treated as just a "flag", and instead its argument then gets treated as a positional argument, and hence saved to `yaml_file_name`. Anyway, not important

Reply:
It can be done a would slightly simplify the code, but the `petablint --help` output would be more confusing, I think.